tox.ini

```diff
@@ -38,6 +38,7 @@ external_wheels =
 setenv =
     COVERAGE_FILE = {env:COVERAGE_FILE:{toxworkdir}/.coverage.{envname}}
     ci: SNOWFLAKE_PYTEST_OPTS = -vvv --color=yes
+    ci2: SNOWFLAKE_PYTEST_OPTS = -vvv --color=yes
```

Comment:
This skips referring to `dist/*py37*` file.

Reply:
Alternatively we could use `pip download` to get the wheel file and then call regular `ci` with it.

Reply:
Yeah, but in this case, the name `dist/*cp37*` won't match anyway because 1.9.1 doesn't have the python version specific wheel. Yet the name of env `ci2` could be better name?